src/codecs/framing/newline_delimited.rs

```diff
@@ -134,9 +134,7 @@ mod tests {
         let mut decoder = NewlineDelimitedDecoder::new_with_max_length(3);
 
         assert_eq!(decoder.decode(&mut input).unwrap().unwrap(), "foo");
-        assert_eq!(decoder.decode(&mut input).unwrap(), None);
         assert_eq!(decoder.decode(&mut input).unwrap().unwrap(), "baz");
-        assert_eq!(decoder.decode(&mut input).unwrap(), None);
```

Comment:
This line should still hold, right? I'd leave it in in that case.

Reply:
Good call 👍